frontend/schemaCMS/src/modules/dataSource/dataSource.sagas.js

```diff
@@ -90,6 +90,7 @@ function* uploadProgressWatcher(channel, id) {
       }
 
       if (data || error) {
+        yield delay(FETCH_LIST_DELAY);
```

Comment:
we could add a test for this delay call

Reply:
I wish to add but it's an inside saga channel and I'm not sure if it's possible to do it.